doc/roadmap.md

```diff
@@ -33,29 +33,32 @@ The following list of features are planned to be addressed in the short term,
 and incorporated into the coming release(s) of the product:
 
 - Allow multiple types in entity to support UNE 178503 requirements ([#3638](https://github.com/telefonicaid/fiware-orion/issues/3638))
-- ~New subscripition modes (create only, update only, delete only and combinations)~ ([#1494](https://github.com/telefonicaid/fiware-orion/issues/1494)
 - Pattern/filter batch updates ([#2389](https://github.com/telefonicaid/fiware-orion/issues/2389))
 - Notification endpoint alias ([#3655](https://github.com/telefonicaid/fiware-orion/issues/3655))
 - Aggregation operations API ([#3816](https://github.com/telefonicaid/fiware-orion/issues/3816))
-- ~Custom notifications: simplifying sending JSON requests~ ([#2560](https://github.com/telefonicaid/fiware-orion/issues/2560))
-
+- Rework commands and deprecate registration API
+- MQTT Retain flag
 
 ## Medium term
 
 The following list of features are planned to be addressed in the medium term,
 typically within the subsequent release(s) generated in the next **9 months**
 after next planned release:
 
-
-- Advanced query language
+- Advanced query language ([4395](https://github.com/telefonicaid/fiware-orion/issues/4395))
 - Signed entities
 - Dynamic / high order attribute values (e.g. an attribute being a sum of two other attributes)
  supported by a Expressions Language - help wanted 
 ([#4004](https://github.com/telefonicaid/fiware-orion/issues/4004)),
 ([#3815](https://github.com/telefonicaid/fiware-orion/issues/3815))
-- Rework commands
 - Service provisioning API (pools, etc.) 
 (based in [#3843](https://github.com/telefonicaid/fiware-orion/issues/3843))
+- Remove registration API
+- Advanced subscription management ([4392](https://github.com/telefonicaid/fiware-orion/issues/4392))
+   - Custom ID subscription
+   - Query Subscrition by fields (endpoint, attributes...)
+   - Configurable statistics consolidation flag (to debug subscriptions) / Debug mode + last 10 subs status?
+   - Flag to disable metadata in subscriptions
```

Comment:
```suggestion
- Advanced subscription management
   - Subscription debug mode (precise statistics consolidation, keep recent history of notifications sent, etc.)
   - Custom ID subscription
   - Query Subscrition by fields (endpoint, attributes...) ([4392](https://github.com/telefonicaid/fiware-orion/issues/4392))
   - Flag to disable metadata in subscriptions
- Remove registration API
```

Create placeholder issues for the 3 sub-items under "Advanced subscriptions management".

Reply:
Done in https://github.com/telefonicaid/fiware-orion/pull/4396/commits/8a566a9c9fa3cc37f8810cf2f9d117334f6b69ae